ansible/roles/stig/tasks/TOSS_04_010370.yml

```diff
@@ -9,21 +9,20 @@
 
 - name: TOSS-04-010370 - There must be no ".shosts" files on The TOSS operating system.
   block:
-    # $ sudo find / -name '*.shosts'
     - name: TOSS-04-010370 - Register *.shosts files
-      ansible.builtin.find:
-        # TODO: Add -xdev flag to avoid crossing filesystems
-        paths: /
-        patterns: '*.shosts'
-        file_type: file
-        recurse: false
+      # as of 2024-12-02, ansible.builtin.find does not have the capability to include the '-xdev' option
+      ansible.builtin.command: "find / -xdev -name '*.shosts'"
```

Comment:
I'm not sure that using the `-xdev` flag is really what we want here... Technically we might want to check the entire system? (yes, I know that doesn't really work in our use case internally due to large systems, but as the standard answer on this STIG it might be the better answer.)